src/components/MenuIcon/MenuIcon.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import type { Meta, StoryObj } from "@storybook/react";
+import { MenuIcon } from './MenuIcon';
+import { action } from '@storybook/addon-actions';
+
+const meta: Meta<typeof MenuIcon> = {
+  component: MenuIcon,
+  tags: ["autodocs"],
+};
+
+export default meta;
+type Story = StoryObj<typeof MenuIcon>;
```

Comment:
Two things:
1. I try to have exports at the bottom of the file so i know where to look for the exports (it's more of a small personal preference that many people also have).
2. We can simplify a bit like this, since meta variable isn't used afterwards:

```suggestion
type Story = StoryObj<typeof MenuIcon>;

export default {
  component: MenuIcon,
  tags: ["autodocs"],
} as Meta<typeof MenuIcon>;
```